pyproject.toml

```diff
@@ -55,9 +55,6 @@ ows = ["OWSLib>=0.20.0", "pillow>=6.1.0"]
 plotting = ["pillow>=6.1.0", "scipy>=1.3.1"]
 test = ["pytest>=5.1.2", "pytest-mpl>=0.11", "pytest-xdist", "pytest-cov", "coveralls"]
 
-[project.scripts]
```

Comment:
In addition to the Why not leaving it also as executable script that is installed with the package, typically named `cartopy_feature_download`? Maybe you must declare a `main` function in the `__main__` module for that.

Reply:
Yeah, I think you're right, we need to define a function that can be called from the script for that. I moved the content in the if block inside a separate `main()` function like you suggested.

Now you can access this script via: `python -m cartopy.feature.download -h` or `cartopy_feature_download gshhs physical --dry-run`

I guess this just brings up the question of whether this installed script is desired or not. I don't have a major preference either way, but would probably slightly lean towards not including it in the scripts section since it is not a critical piece of cartopy. But, it looks like you @stefraynaud were the one that added this as an installation piece, so maybe it is useful for you as a standalone script? Previously, I think it made sense because it was only accessible via a git clone of the repository to access the tools directory, but now you can execute the module which is installed with the package as well.

Reply:
Agree with the first part on this function being useful without git cloning.

My second part was just suggesting to now tell users from wherever you are type: `python -m cartopy.feature.download -h`

But, I'm fine either way with including the install script, so left it in for now.